app-backend/db/src/main/resources/migrations/V80__add_confidence_to_labels.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    annotation_labels
+ADD
+    COLUMN confidence real DEFAULT null;
+
+CREATE INDEX IF NOT EXISTS annotation_labels_confidence_idx ON annotation_labels (confidence);
```

Comment:
We'll need this index for filtering on this field, e.g. if we provide a view of labels with confidence between x and y